### PR TITLE
NOTES-2 : Add the support for Nested Space in Notes Application

### PR DESCRIPTION
**Pull request by acotiuga**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -43,8 +43,8 @@
   </scm>
   <developers>
     <developer>
-      <id>gabrielasmeria</id>
-      <name>Gabriela Smeria</name>
+      <id>acotiuga</id>
+      <name>Alexandru Cotiuga</name>
```
Comment (sdumitriu) [inline pom.xml:51]: You should not remove a past developer, unless there was a copy/paste mistake in the original pom. Instead, append a new `<developer>` tag.

Comment (acotiuga) [inline pom.xml:51]: But what if the past developer just started an idea and created some pages without any working result? Basically this application consisted in a macro call and 3 translation keys. Everything is changed now. How should I proceed in this case? Thanks!

Comment (vmassol) [inline pom.xml:51]: @acotiuga You should not remove past developers unless it was an error. In this case Gabriela did something and it was warranted that she was one of the developers. The fact that the code has been modified doesn't remove the fact that she worked on this extension in the past.

File: src/main/resources/Notes/Translations.fr.xml [hunk 2]
```diff
@@ -1,27 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Enygma2002) [inline src/main/resources/Notes/Translations.fr.xml:1]: Why do you need to remove the French translations page?

Comment (acotiuga) [inline src/main/resources/Notes/Translations.fr.xml:1]: I should have move it tot NoteCode space. Now it's there. Thanks!

**Pull request merged**